src/main/java/uk/gov/hmcts/reform/sscs/service/venue/VenueRpcDetails.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.sscs.service.venue;
+
+import lombok.Getter;
+import uk.gov.hmcts.reform.sscs.model.VenueDetails;
+
+@Getter
+public class VenueRpcDetails {
+
+    private final VenueDetails venueDetails;
+    private final String rpc;
+
+    public VenueRpcDetails(VenueDetails venueDetails) {
+        this.venueDetails = venueDetails;
+        this.rpc = venueDetails.getRegionalProcessingCentre().substring(5);
```

Comment:
Why the substring?

Reply:
as you aware Venue classes are borrowed from HMC codebase, so I assumed they knew what they're doing